frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -316,16 +317,11 @@ const QueueItem = ({
       return false;
     }
     const dateTested = new Date(customDate); // local time, may be an invalid date
-
     // if it is an invalid date
     if (isNaN(dateTested.getTime())) {
       return false;
     }
 
-    // problem: user can manually input future dates and bypass the `max` attribute.
-    // this enforces the max future date to be 1 day from the time of submission, an arbitrary upper bound -- not sure if this is a good idea?
-    const MAX_TEST_DATE = new Date();
-    MAX_TEST_DATE.setDate(MAX_TEST_DATE.getDate() + 1);
     return dateTested > EARLIEST_TEST_DATE && dateTested < MAX_TEST_DATE;
```

Comment:
i suggest in-lining `new Date()` here so we are checking against now, not a cached date

Reply:
Ah, great catch! Fixed in bc2ec5c.